nodejs/week1/src/backend/routes/connection.js

```diff
@@ -0,0 +1,10 @@
+const mysql = require('mysql');
+
+const connection = mysql.createConnection({
+    host: 'localhost',
+    user: 'root',
+    password: 'password',
+    database: 'mealapp'

```

Comment:
I am not sure if your class is using env files to handle credentials but I would def recommend it :) search for .env and how to use it with node (it's pretty much a standard)